core/src/main/scala/kafka/utils/ShutdownableThread.scala

```diff
@@ -27,7 +27,7 @@ abstract class ShutdownableThread(val name: String, val isInterruptible: Boolean
   this.setDaemon(false)
   this.logIdent = "[" + name + "]: "
   val isRunning: AtomicBoolean = new AtomicBoolean(true)
-  private val shutdownLatch = new CountDownLatch(1)
+  val shutdownLatch = new CountDownLatch(1)
```

Comment:
Similar comment regarding making this public. The test only needs the count, so we can expose that as a method.

Reply:
OK. Let me find a way to make its access more constrained and add comment properly.

Reply:
I will update the pull request 3718 to make it `private[server]` and comment that it is `package private for test`.